src/main/java/coloringpage/ImageController.java

```diff
@@ -0,0 +1,93 @@
+package coloringpage;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+public class ImageController {
+
+    private final JLabel originalImageLabel;
+    private final JLabel finalImageLabel;
+    private final Converter converter = new Converter();;
+
+    private BufferedImage lineDrawing;
+
+    public ImageController(JLabel originalImageLabel, JLabel finalImageLabel) {
+        this.originalImageLabel = originalImageLabel;
+        this.finalImageLabel = finalImageLabel;
+    }
+
+    public BufferedImage getFinalImage() {
```

Comment:
why do we need this? if linedrawing is null, wouldnt using that just do the same thing?

Reply:
Whoops, this was a half thought out idea. Meant to add a throws Exception here so that if the image is null an exception is thrown and the final return statement never executes